requirements.txt

```diff
@@ -1,3 +1,4 @@
+landscapes==0.0.10

```

Comment:
Please add this dependency to the `setup.py` as well

Reply:
Done!